zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -10,7 +10,11 @@
 import re
 from zulip_bots.bots.converter import utils
 
-def is_float(value):
+from typing import Any, List
+
+from zerver.lib.type_debug import print_types
```

Comment:
Do we need this import any more?

Reply:
Whoops, my bad. I've believe corrected it now.